client/components/domains/domain-product-price/index.jsx

```diff
@@ -36,26 +37,49 @@ class DomainProductPrice extends React.Component {
 	};
 
 	getDomainPricePopoverElement() {
-		const { price, rule, isFeatured, translate } = this.props;
+		const { price, rule, isFeatured, domain, translate } = this.props;
 
 		let popoverText;
 
 		switch ( rule ) {
 			case 'FREE_DOMAIN':
-				popoverText = translate(
-					'Every WordPress.com site comes with a free address using a WordPress.com subdomain. {{a}}Learn more{{/a}}.',
-					{
-						components: {
-							a: (
-								<a
-									href="https://en.support.wordpress.com/domains/#domain-name-overview"
-									target="_blank"
-									rel="noopener noreferrer"
-								/>
-							),
-						},
-					}
-				);
+				if ( getTld( domain ) === 'blog' ) {
+					popoverText = translate(
+						'Every WordPress.com blog comes with a free .blog address. {{a}}Learn more{{/a}}.',
+						{
+							components: {
+								a: (
+									<a
+										href="https://en.support.wordpress.com/domains/#domain-name-overview"
+										target="_blank"
+										rel="noopener noreferrer"
+										onClick={ event => {
+											event.stopPropagation();
+										} }
+									/>
+								),
+							},
+						}
+					);
+				} else {
+					popoverText = translate(
+						'Every WordPress.com site comes with a free WordPress.com address. {{a}}Learn more{{/a}}.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Every WordPress.com site comes with a free address using a WordPress.com subdomain. {{a}}Learn more{{/a}}.' )` *ES Score: 7*